chapters/6-analysis.tex

```diff
@@ -745,6 +745,29 @@ \subsection{Using {\LaTeX} for dynamic research outputs}
 so papers can be more easily be formatted into their specific layout.
 
 \section{Looking ahead}
+This section discussed the steps needed to take original data
+and create analysis datasets and outputs from it.
+This process created new derived information
+from measurements taken in the field or collected by partners,
+and combined these measures with the research design
+described in the data map and the analysis plan.
```

Comment:
how can information be 'combined' with the research design. perharps organize it for analysis according to the research design, as outlined in the Data Map?

Reply:
Same as above -- this is meant to refer to merging with master datasets according to the other plans, so literally to be merging research variables and measurement variables as we've defined. How to better express that?

Reply:
How about something along the lines of: By combining the variables of interest for the analysis (measurement variables) with the variables describing the analytical design (research variables), you created original datasets ready for analysis.

Reply:
```suggestion
By combining the observed variables of interest for the analysis (measurement variables) 
with the information in the data map describing the study design (research variables), 
you created original datasets ready for analysis.
```